### ISSUE TITLE
update package.js

### ISSUE THREAD

**Issue by imgntn** [sksmatt/NodeJS-Ableton-Piano#1]
should include socket.io and midi in order for the 'npm install' command to work

**Comment by peterheard01** [comment]
yes i thought this and wasn't actually able to get it to run at all locally, any ideas?

**Comment by imgntn** [comment]
two ways:
1. npm install --save socket.io midi
2. you could just clone this fork instead, which seems to work with normal npm install
   https://github.com/robsaunders/nodejs-ableton-piano.git

**Comment by itsMattShull** [comment]
Has anyone else got this to work?  I got it to console log the midi number but don't know how this is supposed to hook into Ableton.  I don't see it in the midi devices.

**Comment by itsMattShull** [comment]
Nevermind I figured it out and forked this repo with better instructions and all the bugs fixed: https://github.com/derekshull/node-midi-controller
